### ISSUE TITLE
is cropping implemented correct? 

### ISSUE THREAD

**Issue by HaiyongJiang** [NVIDIA/flownet2-pytorch#22]
Hi, I found the random cropping is implemented so that each call of the function will crop different regions(datasets.py->StaticRandomCrop). It is unreasonable to me. For each example, we should use random cropping but the left/right/flow image should be cropped in the same way.

**Comment by fitsumreda** [comment]
you are right, there is a bug in the cropping logic. Which could be problematic when training with FlyingThings3D datasets.
For the FlyingChairs, this wouldn't have affected things as the full image size is the crop_size
we'll fix and update the cropping logic. thanks.

**Comment by fitsumreda** [comment]
updated.
